src/Stratis.Bitcoin/Consensus/ConsensusManager.cs

```diff
@@ -610,11 +608,17 @@ private async Task RewindPartiallyConnectedChainAsync(ChainedHeader lastValidate
 
             while (currentTip.Height < current.Height)
```

Comment:
This method looks to me that it is exactly the same as the method above, can't we just use one method and deleted the other one?

Reply:
fixed